editor-components/feedback/feedbackmodal1/feedbackmodal1.tsx

```diff
@@ -0,0 +1,268 @@
+                      value: "Text Area",
+                      additionalParams: {
+                        selectItems: ["Text", "E-mail", "Number", "Text Area"]
+                      }
+                    },
+                    {
+                      type: "string",
+                      key: "type_error_message",
+                      displayer: "Type error message",
+                      value: ""
+                    },
+                  ]
+                },
+              ]
+            }
+          ]
+        }
+      ]
+    });
+
+    this.addProp({
+      type: "array",
+      key: "emojiProps",
+      displayer: "Emojis",
+      value: [
+        {
+          type: "icon",
+          key: "emoji-1",
+          displayer: "Very Bad",
+          value: "FaRegFrown",
+        },
+        {
+          type: "icon",
+          key: "emoji-2",
+          displayer: "Bad",
+          value: "FaRegMeh",
+        },
+        {
+          type: "icon",
+          key: "emoji-3",
+          displayer: "Neutral",
+          value: "FaRegSmile",
+        },
+        {
+          type: "icon",
+          key: "emoji-4",
+          displayer: "Good",
+          value: "FaRegSmile",
+        },
+        {
+          type: "icon",
+          key: "emoji-5",
+          displayer: "Very Good",
+          value: "FaRegGrin",
+        },
+      ],
+    });
+  }
+
+  validationSchema = object().shape({
+    message: string().min(5, "Min 5 characters!").required("Required"),
+  });
+
+  handleEmojiClick(emojiId: number) {
+    const emojiElements = document.querySelectorAll(`.${styles.feedbackModalEmoji}`);
+    emojiElements.forEach((element) => {
+      element.classList.remove(styles.selected);
+      (element as HTMLElement).style.backgroundColor = "transparent";
+    });
+
+    const selectedElement = document.getElementById(`emoji-${emojiId}`);
+    if (selectedElement) {
+      selectedElement.classList.add(styles.selected);
+    }
+  }
+
+  getName(): string {
+    return "FeedbackModal1";
+  }
+
+  render() {
+    const inputItems = this.castToObject<InputItem[]>("input_items") || [];
+    const placeholder = inputItems[0]?.placeholder || "Add a Comment...";
+
+    const emojiProps = this.getPropValue("emojiProps") as Emoji[] || [];
+
+    return (
+      <div className={this.decorateCSS("feedbackModal")}>
+        <div className={this.decorateCSS("feedbackModalHeader")}>
+          <ComposerIcon
+            name={this.getPropValue("feedback_icon")}
+            propsIcon={{
+              className: `${this.decorateCSS("icon")}`,
+            }}
+          />
+          <h1 className={this.decorateCSS("feedbackModalHeaderh1")}>
+            {this.getPropValue("title")}
+          </h1>
+          <button className={this.decorateCSS("feedbackModalCloseButton")}>&times;</button>
+        </div>
+        <div className={this.decorateCSS("headerContent")}>
+          <h2>{this.getPropValue("headerTitle")}</h2>
+        </div>
```

Comment:
if title prop is empty, don't render this div element.
[read here](https://github.com/Teknodev/composer-tools/blob/main/editor-components/README.md#how-to-check-if-a-string-prop-is-empty)